src/server/ns_turn_server.h

```diff
@@ -203,6 +203,9 @@ struct _turn_turnserver {
   /* Return an HTTP 400 response to HTTP connections made to ports not
      otherwise handling HTTP. */
   vintp respond_http_unsupported;
+
+  /* Set to true on SIGUSR1 */
+  int is_draining;
```

Comment:
This is a bool. Use the bool type from stdbool.h

Reply:
And the true and false keywords from the same header in the places where you change the value

Reply:
I was just following suit with the rest of the code here, bool is not used at all in this struct, see members: fingerprint, rfc5780, external_ip_set, etc.  Are you sure you want me to change this and be different than other members?  I'm happy to change it, if you prefer.

Reply:
My team at work is actively working on converting coturn to use stdbool.h, just slow going due to other things suddenly popping up.

It'd be nice to have fewer places to fix :-)

Reply:
While I agree, I also think that code consistency is important, it's confusing if the same thing is done 2 different ways.

Reply:
It's really very confusing to people to have ints used as bools.

Like, i can't make you, but please consider using bool, i'm just going to change it to bool later (and i'll do it one at a time...).

Reply:
Project maintainers do you have an opinion here?  Consistency vs better typing for new code?  

I really am happy to go either way. :)

Reply:
A lot of code has been converted to bool already. Definitely much clearer code.

Reply:
okay - I will change to bool :)